en/06_Project_Governance/06_Minor_release_policy.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Minor Release Policy
+summary: Outline the minor release lifecycle and support commitment for Silverstripe CMS
+icon: code-branch
+---
+
+# Minor Release Policy
+
+This policy complements the [Silverstripe CMS Major release policy](major_release_policy) by providing guidance on when new minor releases of Silverstripe CMS are shipped.
+
+Note that the release cadence and pre-release time frame are indicative only. We may alter those in response to external constraints.
+
+Our minor release policy is more flexible because minor release upgrades are more straight forward to manage than major release upgrades. 
```

Comment:
```suggestion
Our minor release policy is more flexible because minor release upgrades are more straightforward to manage than major release upgrades. 
```

Reply:
Applied suggestion